public/editor-client/src/defaultTemplates/tempConverters.ts

```diff
@@ -1,77 +1,90 @@
 import {
-    BlockWithThumbs,
-    Categories,
-    Kits,
-    KitType,
-    LayoutTemplateWithThumbs,
-    TempLayouts
+  BlockWithThumbs,
+  Categories,
+  Kits,
+  KitType,
+  LayoutTemplateWithThumbs,
+  StoriesTemplateWithThumbs,
+  TempLayouts,
+  TempTemplate
 } from "../types/DefaultTemplate";
 
 export const tempConvertLayouts = (
-    collections: TempLayouts,
-    thumbUrl: string
-): LayoutTemplateWithThumbs[] => collections.templates.map(
-    ({
-         cat,
-         color,
-         name,
-         pages,
-        styles
-     }) => {
-        const isPro = pages.find(item => item.pro === true)?.pro ?? false
-
-        return {
-            cat,
-            color,
-            name,
-            project: name,
-            pagesCount: pages.length,
-            styles,
-            pro: isPro,
-            thumbnailSrc: `${thumbUrl}/${pages[0].id}.jpg`,
-            thumbnailWidth: pages[0].thumbnailWidth,
-            thumbnailHeight: pages[0].thumbnailHeight,
-            keywords: pages[0].keywords
-        };
-    }
-)
+  collections: TempLayouts,
+  thumbUrl: string
+): LayoutTemplateWithThumbs[] =>
+  collections.templates.map(({ cat, color, name, pages, styles }) => {
+    const isPro = pages.find((item) => item.pro === true)?.pro ?? false;
 
+    return {
+      cat,
+      color,
+      name,
+      project: name,
+      pagesCount: pages.length,
+      styles,
+      pro: isPro,
+      thumbnailSrc: `${thumbUrl}/${pages[0].id}.jpg`,
+      thumbnailWidth: pages[0].thumbnailWidth,
```

Comment:
same here, move `pages[0]` in some const

Reply:
this is Layout converter and will be fixed in Layout PR